### DISCUSSION TITLE
如何打包到docker中运行？

### DISCUSSION THREAD

**Discussion by thinker007** [ttttupup/wxhelper#199] [General]
如题

**Comment by godisboy0** [comment]
docker底层是linux，不是windows，这项目基于windows逆向的，我觉得应该是和docker无缘了

**Comment by ttttupup** [comment]
这种hook需要图形界面的，可以关注一下微软官方的镜像和第三方镜像。另外一种就是完全脱机的，就可以随意了。

**Reply by thinker007** [reply]
支持wine就可以哈，已有类似项目：https://github.com/tom-snow/docker-ComWechat
